apps/SageAccountWeb/package.json

```diff
@@ -59,7 +59,7 @@
     "stream-http": "^3.2.0",
     "timers-browserify": "^2.0.12",
     "tslib": "^2.6.2",
-    "type-fest": "^3.13.1",
+    "type-fest": "^4.21.0",
```

Comment:
Fixed a type bug with some new code. Only breaking change in v4 was removal of a utility type we don't use